tests/.editorconfig

```diff
@@ -5,6 +5,9 @@
 # Code files
 [*.{cs,vb}]
 
+# SA0001: XML comment analysis disabled
+dotnet_diagnostic.SA0001.severity = none
```

Comment:
This removes 4 warnings from the build pipeline related to disabled comment analysis.